### PR TITLE
Upgrade SonarQube to 9.9.1

### PR DESCRIPTION
**Pull request by bashbang**
- Upgraded sonarqube to 9.9.1
- removed no longer needed log4j checks/cleanup
- added documentation for upgrade process.

### CODE REVIEW

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
There are a few references to `8.2.2` that need to be updated to `9.9.1`.  Refer to the PR I've opened against your repo; https://github.com/bashbang/sonarqube/pull/1.

The Docker hub image has been built and pushed already.

**Comment by bashbang** [comment]
@WadeBarnes changes accepted and included in PR

**Review by WadeBarnes — APPROVED** [review]

**Comment by WadeBarnes** [comment]
@bashbang, Thanks.  I'll merge the PR and cut a release once my maintainer access has been restored.

**Pull request merged**
